src/utils/useNumber.ts

```diff
@@ -6,6 +6,14 @@ import { useMemo } from 'react';
 
 const getLocaleToUse = (locale?: string) => (locale === 'gx' ? 'fr' : locale || 'en');
 
+export type NumericParser = {
```

Comment:
Could we move this inside /src/types ? I thought it was a util, but it's just the type

Reply:
For now I'd prefer we keep the src/types specific to data modeling and API
we have a ton of types exported from utils and components that are outside of data model

Reply:
and we have a ton of types exported from services as well...